content/en/real_user_monitoring/browser/setup/server.md

```diff
@@ -32,21 +42,79 @@ The automatic installation method requires that you have the [Datadog Agent][2]
+{{% collapse-content title="Automatic instrumentation (recommended)" level="h5" %}}
 
 To automatically instrument your RUM application:
 
-1. In Datadog, navigate to the [**Digital Experience > Add an Application Page**][2] and select the JavaScript (JS) application type.
-2. Select **Auto-Instrumentation** and **Nginx**.
-3. Set your Session and Session Replay sample rates. See [guidance on configuring sampling][3].
-4. Copy and run the installer command to load the Datadog RUM SDK Injector onto your Nginx module.
-5. After the installer successfully installs the SDK Injector, restart Nginx to begin collecting RUM sessions.
+1. In Datadog, navigate to the [**Digital Experience > Add an Application Page**][1] and select the JavaScript (JS) application type.
+2. Select **Auto-Instrumentation** and **NGINX**.
+3. Set your Session and Session Replay sample rates. See [guidance on configuring sampling][2].
+4. Copy and run the installer command to load the Datadog NGINX Module with the RUM SDK Injector onto NGINX.
+5. After the installer successfully installs the SDK Injector, restart NGINX to begin collecting RUM sessions.
+6. (Optional) To verify the module is successfully injecting the RUM Browser SDK into HTML pages, check the NGINX error logs for relevant messages. The module logs important steps during the injection process. Ensure that NGINX is configured with at least the `INFO` log level with the following:
 
-[1]: https://docs.nginx.com/nginx/admin-guide/dynamic-modules/dynamic-modules/
-[2]: https://app.datadoghq.com/rum/list
-[3]: /real_user_monitoring/guide/sampling-browser-plans/
+   ```javascript
+   error_log <file> info;
+   ```
+
+[1]: https://app.datadoghq.com/rum/list
+[2]: /real_user_monitoring/guide/sampling-browser-plans/
+
+{{% /collapse-content %}}
+
+{{% collapse-content title="Manual instrumentation" level="h5" %}}
+
+### Download the appropriate `.tgz` file
+
+1. Use the `.tgz` file corresponding to your version of NGINX. You can find all the relevant `.tgz` files listed by NGINX version under [Reference](#reference).
+2. Extract the tarball to extract the `ngx_http_datadog_module.so` file. Move it to a location that NGINX has access to (referenced as `<RUM_MODULE_PATH>` in the steps below).
+
+### Update NGINX configuration
+1. The `NGINX.conf` file is usually located in NGINX's configuration directory. Add the following line to load the module:
+
+   ```javascript
+   load_module <RUM_MODULE_PATH>;
+   ```
+
+2. Then in the **http/server/location** section, add the following:
+
+   ```javascript
+   # APM Tracing is enabled by default. The following line disables APM Tracing.
+   datadog_disable;
+   datadog_rum on;
+   datadog_rum_config "v5" {
+     "applicationId" "<DATADOG_APPLICATION_ID>";
+     "clientToken" "<DATADOG_CLIENT_TOKEN>";
+     "site" "<DATADOG_SITE>";
+     "service" "my-web-application";
+     "env" "production";
+     "version" "1.0.0";
+     "sessionSampleRate" "50";
```

Comment:
**💬 suggestion:** ‏What about using 100 here? Is there a reason why we should encourage customers to have a lower sample rate?

```suggestion
     "sessionSampleRate" "100";
```